.idea/dictionaries/quocbao.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
```

Comment:
please use `.gitignore` to exclude unnecessary files

Reply:
This is my intention because I use two laptops